test/views/TellUsAboutYourRulingViewSpec.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import forms.TellUsAboutYourRulingFormProvider
+import models.{DraftId, NormalMode}
+import play.twirl.api.HtmlFormat
+import views.behaviours.ViewBehaviours
+import views.html.TellUsAboutYourRulingView
+
+class TellUsAboutYourRulingViewSpec extends ViewBehaviours {
+
+  private val messageKeyPrefix = "tellUsAboutYourRuling"
+
+  private val form: TellUsAboutYourRulingFormProvider =
+    app.injector.instanceOf[TellUsAboutYourRulingFormProvider]
+
+  private val view: TellUsAboutYourRulingView =
+    app.injector.instanceOf[TellUsAboutYourRulingView]
+
+  private val viewViaApply: HtmlFormat.Appendable  =
+    view(form.apply(), NormalMode, DraftId(1L))(fakeRequest, messages)
+  private val viewViaRender: HtmlFormat.Appendable =
+    view.render(form.apply(), NormalMode, DraftId(1L), fakeRequest, messages)
+  private val viewViaF: HtmlFormat.Appendable      =
+    view.f(form.apply(), NormalMode, DraftId(1L))(fakeRequest, messages)
+
+  "TellUsAboutYourRulingView" - {
+
+    def test(method: String, view: HtmlFormat.Appendable): Unit =
+      s"$method" - {
+        behave like normalPage(view, messageKeyPrefix, "")()
+      }
+
+    val input: Seq[(String, HtmlFormat.Appendable)] =
+      Seq(
+        ".apply"  -> viewViaApply,
+        ".render" -> viewViaRender,
+        ".f"      -> viewViaF
+      )
+
+    input.foreach { case (method, view) =>
+      test(method, view)
+    }
+
+    object Selectors extends BaseSelectors {
+      val textBoxLabel = "#main-content > div > div > form > div > div > label"
+    }
+
+    "view should have correct content" - {
+
```

Comment:
Added some content tests, either we hard code the messages which will break on accidental changes of the messages file or just use the message keys which saves on updating these tests during content changes.

using message keys is easier but then doesn't really test the actual content. Since even if you change the content of the keys under test these tests would still pass if implemented in that way.